### PR TITLE
Ellipsis menu demo, lang terms and tests

### PR DESCRIPTION
**Pull request by maboelsoudd2l**
a whole bunch of fun stuff packed into this PR, such as:
1. a demo page for the ellipsis menu
2. visual diff tests for ellispsis menu
3. lang terms added

### CODE REVIEW

File: test/d2l-quick-eval/perceptual/d2l-quick-eval-ellipsis-menu.visual-diff.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		'default',
+	].forEach((name) => {
+		it(name, async function() {
+			const rect = await visualDiff.getRect(page, `#${name}`);
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		});
+	});
+
+	it('opened', async function() {
+		await page.click('#opened d2l-quick-eval-ellipsis-menu');
```
Comment (cbusmand2l) [inline test/d2l-quick-eval/perceptual/d2l-quick-eval-ellipsis-menu.visual-diff.js]: Should we be adding a `waitFor` after this click to make sure that the dropdown animation has finished? I'm worried that we're going to have the same problem we were getting with the more button dropdown and the card hover where it was taking the screenshot before animations were done (sometimes).

File: test/d2l-quick-eval/perceptual/d2l-quick-eval-ellipsis-menu.visual-diff.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		</style>
+		<script type="module">
+			import '@brightspace-ui/core/components/typography/typography.js';
+		</script>
+	</head>
+	<body class="d2l-typography">
+		<div id="default">
+			<d2l-quick-eval-ellipsis-menu></d2l-quick-eval-ellipsis-menu>
+		</div>
+		<div id="opened" style="width: 250px;height: 160px;">
```
Comment (AlexBedley) [inline test/d2l-quick-eval/perceptual/d2l-quick-eval-ellipsis-menu.visual-diff.html]: what's with the styling here?

File: demo/d2l-quick-eval/d2l-quick-eval-ellipsis-menu.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+			`;
+			document.body.appendChild($_documentContainer.content);
+		</script>
+	</head>
+	<body class="d2l-typography">
+		<div class="vertical-section-container fixedSize">
+			<h3>d2l-quick-eval-ellipsis-menu demo</h3>
+			<demo-snippet>
+				<template strip-whitespace>
+					<d2l-quick-eval-ellipsis-menu></d2l-quick-eval-ellipsis-menu>
```
Comment (AlexBedley) [inline demo/d2l-quick-eval/d2l-quick-eval-ellipsis-menu.html]: should we put stuff into the menu for the demo purposes?

File: test/d2l-quick-eval/perceptual/d2l-quick-eval-ellipsis-menu.visual-diff.html (hunk 2 above)
Comment (maboelsoudd2l) [reply test/d2l-quick-eval/perceptual/d2l-quick-eval-ellipsis-menu.visual-diff.html]: So the way that the visual diff works is that it takes an image of the `#opened` div. But the dimensions of the `#opened` div is too small to fit the dropdown of the ellipsis menu. so I hard-coded the dimensions to fit the dropdown (its a quick and dirty solution that works. Let me know if you have other ideas).

File: test/d2l-quick-eval/perceptual/d2l-quick-eval-ellipsis-menu.visual-diff.js (hunk 1 above)
Comment (maboelsoudd2l) [reply test/d2l-quick-eval/perceptual/d2l-quick-eval-ellipsis-menu.visual-diff.js]: good call 👍

File: demo/d2l-quick-eval/d2l-quick-eval-ellipsis-menu.html (hunk 3 above)
Comment (maboelsoudd2l) [reply demo/d2l-quick-eval/d2l-quick-eval-ellipsis-menu.html]: Currently, there isn't much input that `d2l-quick-eval-ellipsis-menu` takes in. That might be subject to change as the dismiss menu will show information. But even then, this dropdown won't really change.

**Comment by maboelsoudd2l** [comment]
With lang terms, I've been trying to find a way to test lang terms that is both easy to write, reliable and doesn't add too much time to the testing. I kinda settled for the visual tests in https://github.com/BrightspaceHypermediaComponents/activities/pull/471 

not saying it's the best solution. How would writing unit tests look like though?

**Comment by maboelsoudd2l** [comment]
@AlexBedley are you looking for unit tests for the ellipsis or the lit-localization thing? I may be misunderstanding this.

**Comment by AlexBedley** [comment]
> @AlexBedley are you looking for unit tests for the ellipsis or the lit-localization thing? I may be misunderstanding this.

I meant for your code that made all the langterm changes... or did you make them by hand?

**Pull request merged**